src/index.html

```diff
@@ -17,11 +17,319 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/style.css"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <div class="catalog">
+      <header class="header">
+        <div class="header__container">
+          <div class="header__logo">
+            <a href="#home">
+              <img
+                src="images/logo.png"
+                alt="Moyo logo"
+              />
+            </a>
+          </div>
+
+          <nav class="nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a
+                  href="#apple"
+                  class="nav__link is-active"
+                >
+                  Apple
+                </a>
+              </li>
+              <li class="nav__item">
```

Comment:
Add the blank line between the links
```suggestion
              </li>
              
              <li class="nav__item">
```